cmd/cloudflared/access/cmd.go

```diff
@@ -38,10 +39,20 @@ const (
 Add to your {{.Home}}/.ssh/config:
 
 {{- if .ShortLivedCerts}}
-Match host {{.Hostname}} exec "{{.Cloudflared}} access ssh-gen --hostname %h"
+{{- if eq .OS "windows"}}
+  ProxyCommand {{.Cloudflared}} access ssh-gen --hostname %h && ssh -tt %r@cfpipe-{{.Hostname}} >&2 <&1
```

Comment:
I think you want `Match host {{.Hostname}} exec` instead of ProxyCommand here, no?

Reply:
how is that different than `ssh -tt %r@cfpipe-{{.Hostname}}`?

Reply:
My bad, 
but you do need to specify that this proxy is only applying to the hostname in question.


```
Host cfpipe-{{.Hostname}}
  ProxyCommand {{.Cloudflared}} access ssh-gen --hostname %h && ssh -tt %r@cfpipe-{{.Hostname}} >&2 <&1
```

Reply:
Sorry for the late revert @GreenStage, just pushed the change that you mentioned. I wonder if that is good enough?